Rakefile

```diff
@@ -6,8 +6,27 @@ require "rake/testtask"
 
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
-  t.pattern = "test/**/*_test.rb"
-  t.verbose = false
+  t.libs << "lib"
+
+  # Running specific tests with line numbers, like with rails test, is not supported by default in rake.
```

Comment:
**thought:** Just FYI, I'm usually using `m` gem for this. It's pretty minimal and offers a way to run a test with a line number as well.

https://rubygems.org/gems/m

Reply:
Yeah I saw. But other gems do not. I don't like to stack gems to get a test running (also appraisal is present in pbbuilder) so I added this in a couple of gems. I kept m in it so you can keep using it.